views/case-building/reasons-for-appeal/reason-for-appeal.njk

```diff
@@ -1,8 +1,10 @@
 {% set pageTitleName = i18n.pages.wrongHomeOfficeDecision.title %}
 {% from "govuk/components/textarea/macro.njk" import govukTextarea %}
 {% extends "layout.njk" %}
+{% set previousPage = "/appellent-timeline" %}
```

Comment:
Shouldn't we set previousPage in the controller and pass it to the view instead of setting it here?